custom-words.txt

```diff
@@ -2810,4 +2810,8 @@ Hnsw
 Piqd
 Reranked
 Reranking
-contentsafety
\ No newline at end of file
+contentsafety
+relro
+RELRO
+runpath
+rpath
```

Comment:
Hm. Without having yet read the jsons, I have to wonder, are these words? Do we need them to be?

Reply:
I think it would make more sense to add them under cSpell than in custom-words, if they're just domain-specific terms for your RP, let me show you how...

In [https://github.com/Azure/azure-rest-api-specs/blob/main/cSpell.json](https://github.com/Azure/azure-rest-api-specs/blob/main/cSpell.json)

look at this as an example [where they had to keep a typo, or non-word, for backCompat]:

        {
            "filename": "**/specification/keyvault/data-plane/Microsoft.KeyVault/**/keyvault.json",
            "words": [
                "Regenerte"
            ]
        },

Reply:
these 3 are domain specific to our firmware analysis, yes. so i would add an entry to cSpell.json and remove them from custom words?

Reply:
Yup, that's the idea